### ISSUE TITLE
VCL: Property TrimTrailingSpaces is not respected during save

### ISSUE THREAD

**Issue by darianmiller** [ideasawakened/DelphiKB#17]
'2020.01.14` 
The TAdvMemo property **TrimTrailingSpaces** is respected during editing, but when the file is saved every line is trimmed.

Feature request added to customer support site:
http://www.tmssoftware.com/site/fr.asp?id=2666 

`procedure TAdvMemoStrings.SaveToFile(const FileName: string);`
...
````
  BeginUpdate;
  for i := 0 to Count - 1 do
  begin
    tr := TrimRight(Strings[i]);
    if Strings[i] <> tr then
      Strings[i] := tr;
  end;
  EndUpdate;

````

**Comment by darianmiller** [comment]
Temporary fix implemented - their code edited and these manual changes documented and will be reapplied on component updates.

**Comment by darianmiller** [comment]
Verified fixed in their latest release v10.1.6.0
